gemrb/GUIScripts/bg2/CharGenCommon.py

```diff
@@ -23,6 +23,7 @@
 import GUICommon
 import CommonTables
 from ie_restype import RES_BMP
+import Spellbook
```

Comment:
Please move it up, so it's with its comrades.

Reply:
Made it reunite with their comrades.